app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1115,6 +1120,44 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         startActivity(AddWidgetInstructionsActivity.intent(context), options)
     }
 
+    override fun onDismissDialog() {
+        val configuration = viewModel.ctaViewState.value?.cta ?: return
```

Comment:
Should this logic be moved to the `ViewModel` in a method called `onDialogDismissed()`?. That way it could also be tested

Reply:
This is now done. In order to make it work I had to create two extra commands specific for DaxDialog actions that don't trigger the `cancelAllAnimations` function.